cmd/cosign/main.go

```diff
@@ -35,7 +36,9 @@ import (
 
 func main() {
 	// Fix up flags to POSIX standard flags.
-	ctx := context.Background()
+	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt)
```

Comment:
this is the main fix

Reply:
this should also look for `SIGTERM` too, since we are distributing cosign in a container form factor